exasol_script_languages_container_ci/lib/common.py

```diff
@@ -0,0 +1,67 @@
+import json
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Callable, Iterable
+from inspect import cleandoc
+
+import docker
+from git import Repo
+
+
+def _get_docker_images():
+    docker_client = docker.from_env()
+    exa_images = []
+    try:
+        exa_images = [str(img) for img in docker_client.images.list() if "exa" in str(img)]
```

Comment:
Not sure if filtering only for exa is a good idea, maybe we should filter for exasol

Reply:
Done